### PR TITLE
Update CI to GitHub Actions

### PR DESCRIPTION
**Pull request by HubLot**
Remove travis and move to GitHub Actions for continous integration.

### CODE REVIEW

**Comment by HubLot** [comment]
Python 2.7 and 3.4 has to be dropped from the CI due to deprecated versions: packages can't be installed anymore on those versions

**Comment by pierrepo** [comment]
OK, this is in line with PR #175

**Comment by HubLot** [comment]
Changes complete for GitHub Actions.

Currently, some tests are skipped and doc failed because of the change in the Weblogo import name (stated in PR # 175).
Once resolved, everything run smoothly.

**Comment by pierrepo** [comment]
@HubLot should I merge this PR first (before #175)?

File: .github/workflows/doc.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
```
Comment (gabrielctn) [inline .github/workflows/doc.yml:11]: Why not also 3.6 here ?

**Comment by HubLot** [comment]
@pierrepo You should merge #175 first and then this one. Otherwise, a merged conflict will arise about the travis file.

File: .github/workflows/doc.yml (hunk 1 above)
Comment (HubLot) [reply .github/workflows/doc.yml:11]: For the build of documentation, I don't think it's necessary to test against all versions of Python.
From what I read, Readthedocs use python 3.7 to produce the documentation. I just add 3.8 for the future but I could have just test 3.7

Comment (gabrielctn) [reply .github/workflows/doc.yml:11]: I see, thanks for the answer

**Comment by pierrepo** [comment]
OK. So let's try not to break the internet then!

**Comment by pierrepo** [comment]
CI from Travis has been disabled.

**Pull request merged**